### ISSUE TITLE
regression error in test_pull1304

### ISSUE THREAD

**Issue by robertoostenveld** [fieldtrip/fieldtrip#1353]
this is the test script that goes with #1304

Looking at the rather short test script, I see that the output of `ft_conenctivityanalysis` is this 

```
Data_stat = 

        label: {3×1 cell}
       dimord: 'chan_chan_freq'
    plvspctrm: [2×2×102 double]
         freq: [1×102 double]
          dof: 32
          cfg: [1×1 struct]
```

which is internally inconsistent, since label has 3 channels, whereas the plvspctrm matches 2 channels. 

I don't really know where this issue originates from, so all of a sudden. @schoffelen do you have an idea?

**Comment by robertoostenveld** [comment]
Oh, I now see that from a whole bunch of recent versions of the dashboard, it is only part of the latest two: and for both it failed. 

````
roboos@mentat001> ls 
016972d  14d2898  1dc9895  31fb95f  4638e5a  48e7f80  596ce22  6f0b995  975f33c  a873f74  c6d8392  e143aeb  e4019ba  f9011ce
05f88d2  154716b  22cbd13  3e6f405  465bce2  561b853  5cf4cf4  71472af  9a7a4b9  bb19088  cc5571b  e32fec3  e91e958  ffd6a03
063af58  1a1d6f8  2a37dc7  4362636  489a563  56a2b8a  6164caf  86d4062  9c1f2a4  c64a83f  d7b3424  e37f1f5  f195093  latest

roboos@mentat001> ls -al */test_pull1304.txt
-rw------- 1 roboos mrphys 4550 Mar 18 19:49 596ce22/test_pull1304.txt
-rw------- 1 roboos mrphys 4550 Mar 18 19:49 latest/test_pull1304.txt
````

Hence it is likely that merging my changes from #1304 actually introduced the problem. The mismatch that we have now seems the opposite from the mismatch that we had before :-(

**Comment by schoffelen** [comment]
I will look at it. I recall that you did something silly with keep the partialed channel in the output label list, but due to the partialisation it should go away.

**Comment by schoffelen** [comment]
This is fixed with PR #1354
